pkg/tfbridge/diff.go

```diff
@@ -268,8 +283,26 @@ func doIgnoreChanges(ctx context.Context, tfs shim.SchemaMap, ps map[string]*Sch
 		en, etf, eps := getInfoFromPulumiName(k, tfs, ps, false)
 		visitPropertyValue(ctx, en, string(k), v, etf, eps, shimutil.IsOfTypeMap(etf), visitor)
 	}
+	return ignoredKeySet
+}
 
-	tfDiff.IgnoreChanges(ignoredKeySet)
+func doIgnoreChanges(
```

Comment:
I couldn't find where this function is actually called (except in tests). Can we remove it or move it into a `*_test.go` file?

Reply:
Yeah moving to _test is good. I left it around to not rewrite the tests.